README.md

```diff
@@ -233,6 +237,42 @@ http {
 }   
 ```
 
+## Alternative Configurations
```

Comment:
Existing users use caching of introspection responses to avoid an introspection request on every API request. For those who want to disable caching, I would like to ensure that this section is accurate.